scripts/convert_to_csv.py

```diff
@@ -1,40 +1,35 @@
 import pyreadr
-import glob, os
+from pathlib import Path
 
 
 errors = []
+need_zip = [
+    "ncdc_normals.rda",
+]
 
-for fn in glob.glob("../data/rda_files/*.rda"):
+rda_dir = Path("../data/rda_files")
+csv_dir = Path("../data/csv_files")
+errors_file = Path("..") / "data" / "errors.txt"
+
+for fn in rda_dir.glob("*.rda"):
+    base_name = fn.name
     try:
-        data = pyreadr.read_r(fn, timezone="PST")
+        data = pyreadr.read_r(str(fn), timezone="PST")

```

Comment:
I forgot to mention: I don't actually know what his `timezone` arg does in detail. I just put PST because that's where I live. If we have to put *something* then maybe the UTC (+0) timezone is a better choice?

Reply:
I actually don't know too but I tried Googling around now and from what I learnt, I do not think it is an important arg to add. We can totally omit it from the function call.